reactfrontend/src/components/addAssetForm/addAssetForm.js

```diff
@@ -162,19 +173,19 @@ class AddAssetForm extends Component {
   render() {
     let successMessage = null
     let failMessage = null
-    if(this.state.showMessage == "successMessage") {
+    if(this.state.message == "successMessage" && this.state.showMessage == true) {
```

Comment:
'message' and 'ShowMessage' could be mixed up when someone is reading. Consider trying something that reflects what each message does?

Reply:
Fixed